src/rust/dbv2/migrations/2023-12-01-180017_healthcheck/up.sql

```diff
@@ -0,0 +1,98 @@
+-- Your SQL goes here
+CREATE VIEW
+  api.user_history_last_indexed_txn AS
+SELECT
+  *
+FROM
+  aggregator.user_history_last_indexed_txn;
+
+
+GRANT
+SELECT
+  ON api.user_history_last_indexed_txn TO web_anon;
+
+
+CREATE VIEW
+  api.candlesticks_last_indexed_txn AS
+SELECT
+  *
+FROM
+  aggregator.candlesticks_last_indexed_txn;
+
+
+GRANT
+SELECT
+  ON api.candlesticks_last_indexed_txn TO web_anon;
+
+CREATE VIEW api.data_status AS
+SELECT
+    CURRENT_TIMESTAMP AS current_time,
+    processor_status.last_success_version AS processor_last_txn_version_processed,
+    user_history_last_indexed_txn.txn_version AS aggregator_user_history_last_txn_version_processed,
+    array_agg((resolution, candlesticks_last_indexed_txn.txn_version)) AS aggregator_candlesticks_last_txn_version_processed,
+    api.data_is_consistent() AS aggregator_data_passes_simple_consistency_check
+FROM 
+
+CREATE FUNCTION api.data_is_consistent ()
+RETURNS boolean AS $$
+    SELECT (
+        SELECT COALESCE((
+            SELECT SUM(total_filled)
+            FROM api.orders
+        ) = (
+            SELECT SUM(size)
+            FROM fill_events
+            WHERE txn_version <= (
+                SELECT txn_version
+                FROM api.user_history_last_indexed_txn
+            )
+        ), true)
+    ) AND COALESCE((
+        WITH last_candlesticks AS (
+            SELECT DISTINCT ON (market_id, resolution)
+                volume,
+                start_time,
+                resolution,
+                market_id
+            FROM api.candlesticks
+            ORDER BY market_id, resolution, start_time DESC
+        )
+        SELECT
+            SUM(CASE WHEN ok THEN 0 ELSE 1 END) = 0
+        FROM (
+            SELECT SUM(size*price)/2 = volume AS ok
```

Comment:
Dividing by two doesn't necessarily solve deduping because sometimes fill events are not duplicated

How about checking fille events where maker address is emit address?